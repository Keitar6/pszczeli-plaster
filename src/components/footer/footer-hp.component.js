import "./footer-hp.styles.css";

export const FooterHP = () => {
  return <div className="footerHp-container">FOOTER ON HOMEPAGE</div>;
};
