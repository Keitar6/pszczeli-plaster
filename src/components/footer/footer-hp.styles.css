.footerHp-container{

    border: 0.1rem solid red
}