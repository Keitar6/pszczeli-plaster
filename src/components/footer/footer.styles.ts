import styled from "styled-components";
import { Colors } from "../../global.styles";

export const FooterContainer = styled.div`
  background-color: ${Colors.primary};
  display: flex;
  justify-content: center;
`;

export const FootersContent = styled.div`
  display: inline-flex;
  flex-direction: row;
  gap: 10rem;
  flex-wrap: wrap;
  padding: 0 1rem 0 1rem;
`;
