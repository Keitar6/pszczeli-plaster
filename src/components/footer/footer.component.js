import { FooterContainer, FootersContent } from "./footer.styles";

import { Informacje } from "./footers components/footer-Informacje/footer-Informacje.component";
import { MojeKonto } from "./footers components/footer-Moje_Konto/footer-Moje_Konto.component";
import { ONas } from "./footers components/footer-O_Nas/footer-O_Nas.component";
import { PlatnosciIDostawa } from "./footers components/footer-Platnosc_I_Dostawa/footer-Platnosc_I_Dostawa.component";

export const Footer = () => {
  return (
    <FooterContainer>
      <FootersContent>
        <PlatnosciIDostawa />
        <Informacje />
        <ONas />
        <MojeKonto />
      </FootersContent>
    </FooterContainer>
  );
};
