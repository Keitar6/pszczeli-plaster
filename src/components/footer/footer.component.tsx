import { FC } from "react";
import { FooterContainer, FootersContent } from "./footer.styles";

import { Info } from "./footers components/footerInfo/footerInfo.component";
import { MyAccount } from "./footers components/footerMyAccount/footerMyAccount.component";
import { AboutUs } from "./footers components/footerAboutUs/footerAboutUs.component";
import { PayDeliver } from "./footers components/footerPayDeliver/footerPayDeliver.component";
import { selectCurrentUser } from "../../store/userReducer/user.selector";
import { useAppSelector } from "../../hooks/hooks";

export const Footer: FC = () => {
  const currentUser = useAppSelector(selectCurrentUser);

  return (
    <FooterContainer>
      <FootersContent>
        <PayDeliver />
        <Info />
        <AboutUs />

        {currentUser ? <MyAccount /> : null}
      </FootersContent>
    </FooterContainer>
  );
};
