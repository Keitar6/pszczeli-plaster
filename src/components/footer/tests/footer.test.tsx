import { Footer } from "../footer.component";
import ComponentsRenderer from "react-test-renderer";
describe("Footer", () => {
  test("footerAboutUs", () => {
    const text = ComponentsRenderer.create(<Footer></Footer>).toJSON();
    expect(text).toMatchSnapshot();
  });
});
