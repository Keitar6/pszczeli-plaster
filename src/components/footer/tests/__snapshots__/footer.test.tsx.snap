// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer footerAboutUs 1`] = `
<div
  className="sc-papXJ NFyYL"
>
  <div
    className="sc-iBkjds sc-jqUVSM gftbFx hbFiEh"
  >
    <div
      className="sc-dmRaPn cpFxFl"
    >
      <h4
        className="sc-hHLeRK cutfUm"
      >
        Płatności i dostawa
      </h4>
      <div
        className="sc-dIouRR hrcPPE"
      >
        <div
          className="sc-ftvSup cvUJgU"
        >
          Czas realizacji zamówień
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Faktury i paragony
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Opcje dostawy
        </div>
      </div>
    </div>
    <div
      className="sc-crXcEl dxxyJK"
    >
      <h4
        className="sc-iqcoie iXwxym"
      >
        Informacje
      </h4>
      <div
        className="sc-kDDrLX eehIRC"
      >
        <div
          className="sc-ftvSup cvUJgU"
        >
          Warunki reklamacji
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Zwroty towaru
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Regulamin
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Polityka cookies
        </div>
      </div>
    </div>
    <div
      className="sc-fEOsli gRMNcJ"
    >
      <h4
        className="sc-fnykZs bhbXbt"
      >
        O Nas
      </h4>
      <div
        className="sc-hAZoDl jBJKdL"
      >
        <div
          className="sc-ftvSup cvUJgU"
        >
          O nas
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Sklepy stacjonarne
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Współpraca
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Media społecznościowe
        </div>
        <div
          className="sc-bjUoiL jpliOr"
        >
          <a
            className="sc-idiyUo jhdLKE"
          >
            <span />
          </a>
          <a
            className="sc-idiyUo jhdLKE"
          >
            <span />
          </a>
          <a
            className="sc-idiyUo jhdLKE"
          >
            <span />
          </a>
          <a
            className="sc-idiyUo jhdLKE"
          >
            <span />
          </a>
        </div>
      </div>
    </div>
    <div
      className="sc-ksZaOG irkxga"
    >
      <h4
        className="sc-breuTD hPlSIS"
      >
        Moje Konto
      </h4>
      <div
        className="sc-evZas gHCSVc"
      >
        <div
          className="sc-ftvSup cvUJgU"
        >
          Historia zamówień
        </div>
        <div
          className="sc-ftvSup cvUJgU"
        >
          Ustawienia Konta
        </div>
      </div>
    </div>
  </div>
</div>
`;
